src/js/dashboard.js

```diff
@@ -0,0 +1,52 @@
+$(function() {
+  var tabList = $('#tab-list');
+  var tabDiv = $('.tab');
```

Comment:
please consider about using createElement.(see http://stackoverflow.com/questions/494143/creating-a-new-dom-element-from-an-html-string-using-built-in-dom-methods-or-pro)